modules/ding_provider/connie_search/src/ConnieTingObjectCollection.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ * The ConnieTingObjectCollection class.
+ */
+
+namespace Connie\Search;
+
+use Ting\TingObjectCollectionInterface;
+
+class ConnieTingObjectCollection implements TingObjectCollectionInterface {
+
+  /**
+   * @var \Ting\TingObjectInterface[]
+   */
+  protected $objects;
+
+  /**
+   * ConnieObjectCollection constructor.
+   *
+   * @param \Ting\TingObjectInterface[] $objects
+   *   The objects in the collection.
+   */
+  public function __construct($objects) {
```

Comment:
How about type hinting for an array here? Hardly perfect but better than nothing.